frontend/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap

```diff
@@ -67,24 +67,22 @@ exports[`MineNoticeOfWorkTable renders properly 1`] = `
     dataSource={
       Array [
         Object {
-          "date": "Aug 14 2019",
-          "key": "07e801a0-fa33-4c3b-abcc-ac6df628d483",
+          "date": "N/A",
+          "key": undefined,
```

Comment:
this looks like mocks needs to be updated.

Reply:
Good catch; will update

Reply:
Updated